src/test/java/net/javadiscord/javabot/util/IndentationHelperTest.java

```diff
@@ -0,0 +1,44 @@
+package net.javadiscord.javabot.util;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+/**
+ * Test for the {@link IndentationHelper} class.
+ */
+public class IndentationHelperTest {
+    /**
+     * Tests the {@link IndentationHelper#formatIndentation(String, IndentationHelper.IndentationType)} method
+     */
+    @Test
+    public void testFormatIndentation() {
+        String[] unformatted = null;
+        String[] formatted = null;
+        try {
+            unformatted = Files.readString(Path.of(IndentationHelper.class.getResource("/Unformatted Strings.txt").toURI())).split("----");
+            formatted = Files.readString(Path.of(IndentationHelper.class.getResource("/Formatted Strings.txt").toURI())).split("----");
+        } catch (NullPointerException | URISyntaxException e) {
+            fail("Files to run the test not present");
```

Comment:
Why not just throw the exception and let JUnit mark the test case as errored?

Reply:
I like to give a concrete reason why this test failed. I will pass the throwable as well to provide more information